app/packages/multimodal/src/views/episode/transforms/TransformGraphCanvas.module.css

```
@@ -0,0 +1,118 @@
+}
+
+.canvas {
+  background: var(--color-content-bg-background);
+  flex: 1;
+  min-height: 180px;
+  overflow: hidden;
+  position: relative;
+  touch-action: none;
+}
+
+.canvas :global(.react-flow) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)